odhargo/odhargo/base/argo-server-route.yaml

```diff
@@ -1,11 +1,10 @@
 kind: Route
 apiVersion: route.openshift.io/v1
 metadata:
-  name: argo-portal
```

Comment:
This may be a potential issue and would need remapping in odh-dashboard. I've renamed the `argo-portal` route to `argo-server` since both the [`Service`](https://github.com/opendatahub-io/odh-manifests/pull/208/files#diff-5fe25086066c4f4201a497a261b3214ce38a12f9ac1ae29a4669b5f729d8b838R201) and [`Deployment`](https://github.com/opendatahub-io/odh-manifests/pull/208/files#diff-5fe25086066c4f4201a497a261b3214ce38a12f9ac1ae29a4669b5f729d8b838R226) in the upstream manifests (as well as in the renders here) are using the `argo-server` name, so we're rather consistent.

Reply:
Resolved via https://github.com/opendatahub-io/odh-dashboard/pull/37